### PR TITLE
CI: communicate status of core-ci with core

### PR DESCRIPTION
**Pull request by LMMilliken**
This pr adds a step at the start and end of the `remote-ci` workflow that updates a comment in the pr of the branch that is being tested, updating the progress and outcome of the job.

This is what will be commented on the pr in core that triggers the job:

Upon pushing to the pr, before client receives the message:
image

Once client receives the message and starts testing:
image

Once testing is complete:
image

---

- [x] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

**Review by bwanglzu — APPROVED** [review]
LGTM! Let's merge and make a run

**Review by guenthermi — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,10 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add folder for example CSV files ([#663](https://github.com/jina-ai/finetuner/pull/663))
 
+- Add communication between remote-ci job and the pr that triggered it ([#642](https://github.com/jina-ai/finetuner/pull/642))
```
Comment (guenthermi) [inline CHANGELOG.md:15]:
```suggestion
- Add communication between remote-ci job and the pr that triggered it. ([#642](https://github.com/jina-ai/finetuner/pull/642))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,10 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add folder for example CSV files ([#663](https://github.com/jina-ai/finetuner/pull/663))
```
Comment (guenthermi) [inline CHANGELOG.md:13]:
```suggestion
- Add folder for example CSV files. ([#663](https://github.com/jina-ai/finetuner/pull/663))
```

**Review by guenthermi — COMMENTED** [review]
only this two minor things

**Pull request merged**